operator/v1/types_network.go

```diff
@@ -411,6 +411,9 @@ type OVNKubernetesConfig struct {
 	// gatewayConfig holds the configuration for node gateway options.
 	// +optional
 	GatewayConfig *GatewayConfig `json:"gatewayConfig,omitempty"`
+	// egressIPConfig holds the configuration for EgressIP options.
+	// +optional
+	EgressIPConfig *EgressIPConfig `json:"egressIPConfig,omitempty"`
```

Comment:
Why does this need to be a pointer? In general that harms discoverability for end users

Reply:
This harms discoverability for end users as I've previously said, so we should endeavour with new additions to be compliant with the latest API conventions. We don't repeat the sins of our past.

xRef to API conventions FAQ: https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#my-proposed-design-looks-like-an-existing-api-we-have-why-am-i-being-told-that-it-must-be-changed